style/images/open-file.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15 15" fill="none" />
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15 15" fill="none">
```

Reply:
yikes, good catch. that's what I get for cleaning up the file after I made the GIF. I  was also missing a `/>` in one of the paths.